pandasai/responses/streamlit_response.py

```diff
@@ -18,7 +18,9 @@ def format_plot(self, result) -> None:
         try:
             image = mpimg.imread(result["value"])
         except FileNotFoundError as e:
-            raise FileNotFoundError(f"The file {result['value']} does not exist.") from e # noqa: E501
+            raise FileNotFoundError(
+                f"The file {result['value']} does not exist."
+            ) from e  # noqa: E501
```

Comment:
Resolving PEP lines length violations is a good practice, but then there is no need for E501 to be suppressed:
```python
) from e
```

Reply:
Yeah.. that was done by black python formatter.. i didn't noticed this. I will remove it .Thanks  👍